tests/unit/currencyFormat.spec.ts

```diff
@@ -24,6 +24,12 @@ describe('CurrencyFormat', () => {
         expect(new CurrencyFormat({ currency: 'EUR', precision: 0 })).toEqual(expect.objectContaining({ minimumFractionDigits: 0, maximumFractionDigits: 0 }))
       })
 
+      it('should work with a custom precision range', () => {
+        expect(new CurrencyFormat({ currency: 'EUR', precision: {min: 0, max: 4} })).toEqual(
```

Comment:
There should be spaces between the curly braces. You can fix code style issues with `yarn lint:fix`.

Reply:
Fixed, thanks!